core/lib/spree/testing_support/factories/shipping_method_factory.rb

```diff
@@ -10,6 +10,12 @@
       end
     end
 
+    after(:create) do |shipping_method, evaluator|
+      if shipping_method.stock_locations.empty?
+        shipping_method.stock_locations << (Spree::StockLocation.all.to_a)
```

Comment:
Also, is the `#to_a` needed? I think rails actually calls it internally. That's how it does the DB request at the end. Calling `#to_a` here just hands you back an `Array` object instead of an `ActiveRecord::Relation` object. I would always prefer an AR object over a ruby object.